apps/src/blockly/addons/cdoFieldAngleTextInput.ts

```diff
@@ -1,44 +1,117 @@
-import {FieldAngle} from '@blockly/field-angle';
+import GoogleBlockly from 'blockly/core';
 
-import {CLOCKWISE_TURN_DIRECTION} from '../constants';
+import {ExtendedBlockSvg} from '../types';
+
+import CdoAngleHelper from './cdoAngleHelper';
 
 interface AngleTextInputOptions {
   directionTitle: string; // Ex. 'DIR'
   direction: string; // Ex. 'turnRight'
```

Comment:
will the direction always be some specific string like `turnRight` or `turnLeft`? If so, you can make this a more specific type with this syntax:
```
direction: 'turnRight' | 'turnLeft';
```